test/common/upstream/least_request_load_balancer_fuzz_test.cc

```diff
@@ -0,0 +1,37 @@
+#include <memory>
+
+#include "test/common/upstream/least_request_load_balancer_fuzz.pb.validate.h"
+#include "test/common/upstream/load_balancer_fuzz_base.h"
+#include "test/fuzz/fuzz_runner.h"
+#include "test/test_common/utility.h"
+
+namespace Envoy {
+namespace Upstream {
+
+DEFINE_PROTO_FUZZER(const test::common::upstream::LeastRequestLoadBalancerTestCase input) {
+  try {
+    TestUtility::validate(input);
+  } catch (const ProtoValidationException& e) {
+    ENVOY_LOG_MISC(debug, "ProtoValidationException: {}", e.what());
+    return;
+  }
+
+  std::unique_ptr<LoadBalancerFuzzBase> load_balancer_fuzz =
```

Comment:
Can LoadBalancerFuzzBase be created on the stack? If so, avoiding the heap allocation will save you a few cycles.

Reply:
Hey, thanks for the comments! Sorry for the confusion, but this pr and #13425 are both based on a base class currently going through review passes in #13424. I'll address all your comments with a commit to that one, and close these two PR's for now.

Reply:
Switched to stack.